presto-docs/src/main/sphinx/installation/deployment.rst

```diff
@@ -2,6 +2,38 @@
 Deploying Presto
 ================
 
+.. _requirements:
+
+Requirements
+------------
+
+.. _requirements-linux:
+
+Linux Operating System
+^^^^^^^^^^^^^^^^^^^^^^
+
+* 64-bit required
+* newer release preferred, especially when running on containers
+
+.. _requirements-java:
+
+Java Runtime Environment
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+* 64-bit required
+* version 11 recommended
```

Comment:
Let's combine these

> version 11 recommended (higher versions are less tested)

Reply:
done

Reply:
I prefer using , instead of () personally .. but in the interest of getting this merged..